tutorials/sphinx-tutorials/multiagent_competitive_ddpg.py

```diff
@@ -3,6 +3,12 @@
 ===========================================================================
 **Author**: `Matteo Bettini <https://github.com/matteobettini>`_
 
+.. note::
+   If you are interested in benchmarking Multi-Agent Reinforcement Learning (MARL) in
+   TorchRL, there exists an external facebookresearch project
+   `BenchMARL <https://github.com/facebookresearch/BenchMARL>`__ that can help you
+   train and compare MARL sota-implementations, tasks, and models using TorchRL!
```

Comment:
What do we want from these headers exactly? When I read this it feels promotional and not a valuable info for anyone who wants to get proficient with the MARL API, which is the goal of the tuto (ie. it's unclear how this can be a next step for users who are reading this tuto).

I'd stick to a very concise, simple sentence like
```suggestion
   The `BenchMARL <https://github.com/facebookresearch/BenchMARL>`__ library provides state-of-the-art implementations of MARL algorithms.
```